pytest.ini

```diff
@@ -1,2 +1,10 @@
 [pytest]
-addopts = --doctest-glob='scrapinghub/*.py'
+addopts =
+    --doctest-glob='scrapinghub/*.py'
+    --ignore=tests/client/test_items.py
+    --ignore=tests/client/test_job.py
+    --ignore=tests/client/test_logs.py
+    --ignore=tests/client/test_requests.py
+    --ignore=tests/client/test_samples.py
+    --ignore=tests/client/test_spiders.py
+    --ignore=tests/hubstorage/test_retry.py
```

Comment:
Why are we ignoring test files?

If the issue is that a newer pylint includes new checks, I think it would be better to configure Pylint to ignore those checks until we get around to address them. But checks that test files already pass should continue to be checked for new code, so ignoring the whole files is, in my opinion, not the best approach.